CitationStyleLanguageSettingsForm.inc.php

```diff
@@ -34,7 +34,9 @@ public function __construct($plugin) {
 	/**
 	* @copydoc Form::init
 	*/
-	public function initData($request) {
+	public function initData() {
+		$application = PKPApplication::getApplication();
+		$request = $application->getRequest();
```

Comment:
Thanks @crism. I think we want to use `Application` instead of `PKPApplication` in most cases, and you can go straight to the request from there:

```
$request = Application::getRequest()
```

If you're able to make this change, that'd be great. If you've got a million other things to manage, no worries, I'm happy to merge it as-is.

Reply:
Changed.